components/rmrk/Create/DropUpload.vue

```diff
@@ -39,10 +44,12 @@
 <script lang="ts">
 import { Component, Emit, Prop, Ref, Vue, Watch } from 'nuxt-property-decorator'
 import Tooltip from '@/components/shared/Tooltip.vue'
+import ModelMedia from '~~/components/media/type/ModelMedia.vue'
```

Comment:
It doesn't work for me either

![Screenshot 2022-11-09 at 13-27-39 KodaDot Low fees and low carbon minting KodaDot - Kusama NFT Market Explorer](https://user-images.githubusercontent.com/9987732/200830522-fd7cf908-ca41-471b-b53d-1d01f77bfc06.png)

Reply:
no.
it doesn't work yet
problem is chrome doesn't seem to support model/* types
so in places where we use DropUpload without accept prop it works (rmrk/create nft)

but where we want to restrict allowed file type such as in BaseCollectionForm it doesn't, because chrome doesn't work with model/* types
(rmrk/create collection, bsx/ create collection)

it works well in Firefox

I'm open to suggestions on how to overcome this issue please

Reply:
@roiLeo ok then, if that's the expected behavior then all good